src/gui/Window.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Config.h"
 #include "EntryTree.h"
 #include "FileViewer.h"
+#include "FilesystemSearchProvider.h"
```

Comment:
SAPP is a single header library, it's one of the caveats of having a single header library, the only reason it has a .cpp file is because CMake WILL complain about not knowing the linker type. I don't know why.

Reply:
If you don't need SAPP to have a cpp file make it an interface library